xml/_types.ts

```diff
@@ -22,10 +22,12 @@ export type xml_node = {
   [key: string]: unknown
 }
 
+export type v1_x = `1.${number}`
```

Comment:
Can you add a `/** */` comment on this so it has a description for documentations ?

Reply:
I think I'm going to change the approach, but otherwise I'd be happy to 😉